packages/web/src/features/clubs/components/ClubListGrid.tsx

```diff
@@ -12,6 +12,7 @@ import type { ClubCardProps } from "@sparcs-clubs/web/features/clubs/components/
 
 interface ClubListGridItemProps {
   clubList: Array<ClubCardProps["club"]>;
+  isRegistrationPeriod: boolean;
```

Comment:
요건 optional로 해도 괜찮을듯??? (사실 큰 상관 없음)
등록 기간이 아닐 때에는 중요하지 않아서(?)

```suggestion
  isRegistrationPeriod?: boolean;
```

Reply:
는 뒤쪽애 마이페이지 쪽도 확인해보니까 optional로 바꾸는 것이 나을 것 같네용